build/config.props

```diff
@@ -19,12 +19,12 @@
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">rtm</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.4</ReleaseLabel>
```

Comment:
I wouldn't use preview 4 branding. It might be the last insertion we do before the product goes GA.

Reply:
I see..Thanks!